cernopendata/modules/theme/assets/semantic-ui/js/search/components/Facets.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * -*- coding: utf-8 -*-
+ *
+ * This file is part of CERN Open Data Portal.
+ * Copyright (C) 2021 CERN.
+ *
+ * CERN Open Data Portal is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * CERN Open Data Portal is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CERN Open Data Portal; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307, USA.
+ *
+ * In applying this license, CERN does not
+ * waive the privileges and immunities granted to it by virtue of its status
+ * as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+import React from "react";
+import { BucketAggregation, Toggle } from "react-searchkit";
+
+
+const CODFacets = ({ aggs }) => {
+  return (
+    <>
+      <Toggle
+        title="Availability"
+        label="View ondemand datasets"
+        filterValue={["ondemand", "true"]}
+      />
```

Comment:
![Screenshot 2021-07-09 at 13-14-30 CERN Open Data Portal](https://user-images.githubusercontent.com/4990025/125069883-a4981780-e0b7-11eb-9948-089dd59ca739.png)
For me the label of this component is displayed in two lines. Could be easily fixed with some css

Reply:
What about just keeping `Ondemand datasets`, is it intuitive?

Reply:
The original text, i.e. "include on-demand datasets", would be the clearest.  

Otherwise people may wonder why there is no "availability: online" for example. This might actually be the shortest option:

```
Availability:

[x] online
[ ] ondemand
```

I'll have a comment about the order of facets, e.g. they don't match the current order... So we could perhaps muse and fix it in a separate ticket...